src/style.css

```diff
@@ -0,0 +1,131 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  margin: auto;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin: auto;
+  width: 100%;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  padding-left: 16px;
+}
+
+.card__text-product-code {
+  margin: auto;
+  width: 100%;
+  padding-top: 4px;
+  padding-left: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__reviews-text {
+  padding-right: 16px;
+  padding-top: 18px;
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-text {
+  padding-left: 16px;
+  padding-top: 24px;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-amount {
+  padding-top: 24px;
+  margin-right: 16px;
+  margin-bottom: 16px;
+  width: 50px;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  margin: auto;
+  margin-bottom: 16px;
+  height: 40px;
+  width: 166px;
+
+  font-weight: 700;
+  font-size: 14px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #f3f3f3;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.card__button:hover {
+  max-height: 38px;
+  color: #00acdc;
+  background-color: white;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  padding-left: 12px;
+  padding-top: 16px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
```

Comment:
need to transfer all styles for stars to keep the ability to change the number of active stars by changing the modifier